scripts/homestead.rb

```diff
@@ -410,5 +410,41 @@ def Homestead.configure(config, settings)
             s.args = [settings["ip"]]
             s.privileged = false
         end
+
+        if settings.has_key?("backup") && (Vagrant::VERSION >= '2.1.0' || Vagrant.has_plugin('vagrant-triggers'))
+            dirPrefix = '/vagrant/'
+            settings["backup"].each do |backup|
+                type = backup["type"]
```

Comment:
Backup type should be irrelevant because we created mysql/maria/postgres DBs for everything in `settings["databases"]`